.kibana-plugin-helpers.json

```diff
@@ -1,6 +1,5 @@
 {
-  "styleSheetToCompile": "public/app.scss",
-  "buildSourcePatterns": [
+  "serverSourcePatterns": [
```

Comment:
Minor:  `index.js` can be removed.
Besides, I think the `public` folder can be excluded from the putting into the zip file, as the `target` folder contains the rendered pages.

Reply:
Removed `index.js`.